lib/models/game.rb

```diff
@@ -0,0 +1,24 @@
+require 'errors/game_over_error'
+
+module RubyConnect
+  class Game
+    attr_accessor :board, :winning_move
+
+    def initialize(player1, player2)
+      @board = Board.new
+      @winning_move = false
+      @player1 = player1
+      @player2 = player2
+    end
+
+    def done?
+      board.full? || winning_move
+    end
+
+    def make_move(column, color)
+      raise GameOverError if done?
```

Comment:
Is a having a `winning_move` an exceptional event? Or in other words, is an exception the best way to communicate this information to the client (user interface for the game)?